### PR TITLE
kernel: bump 5.4 to 5.4.124

### PR DESCRIPTION
**Pull request by graysky2**
Manually rebased:
```
  generic/hack-5.4/662-remove_pfifo_fast.patch
```
All other patches automatically rebased.

Build system: x86_64
Build-tested: ipq806x/R7800
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

### CODE REVIEW

File: target/linux/generic/hack-5.4/662-remove_pfifo_fast.patch [hunk 1]
```diff
@@ -1,224 +0,0 @@
...
-Date: Fri, 7 Jul 2017 17:23:42 +0200
-Subject: net: sched: switch default qdisc from pfifo_fast to fq_codel and remove pfifo_fast
-
-Signed-off-by: Felix Fietkau <nbd@nbd.name>
----
- net/sched/sch_generic.c | 140 ------------------------------------------------
- 1 file changed, 140 deletions(-)
-
---- a/net/sched/sch_generic.c
-+++ b/net/sched/sch_generic.c
```
Comment (hauke) [inline target/linux/generic/hack-5.4/662-remove_pfifo_fast.patch:12]: Why did you remove this patch?
The upstream changes are not looking like they integrated this functionality:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/net/sched/sch_generic.c?id=v5.4.124&id2=v5.4.123

Comment (graysky2) [reply target/linux/generic/hack-5.4/662-remove_pfifo_fast.patch:12]: Ah, I see this now.  My mistake was believing the quilt output:
```
% quilt push
Applying patch generic-hack/662-remove_pfifo_fast.patch
patching file net/sched/sch_generic.c
Hunk #1 FAILED at 595.
1 out of 1 hunk FAILED -- rejects in file net/sched/sch_generic.c
Patch generic-hack/662-remove_pfifo_fast.patch can be reverse-applied
```
I see now that it needed a refresh.  I will re-build/verify and fpush when good.

File: target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch [hunk 2]
```diff
@@ -20,22 +20,22 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
...
 +#include <linux/module.h>
  #include <linux/of_platform.h>
  #include <linux/of_address.h>
  
-@@ -24,6 +25,7 @@
+@@ -25,6 +26,7 @@
  #include "common.h"
  
  __iomem void *rt_sysc_membase;
 +EXPORT_SYMBOL(rt_sysc_membase);
```
Comment (hauke) [inline target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch:35]: This export was backported and the include is probably also not needed nay more.

Comment (graysky2) [reply target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch:35]: Is your recommendation to simply delete the include and rebase?

**Comment by hauke** [comment]
Thank you for the patch, I applied it with the modified patches-5.4/0048-asoc-add-mt7620-support.patch  change to master.

File: target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch (hunk 2 above)
Comment (hauke) [reply target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch:35]: I removed this part myself before pushing.
